src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculable {
```

Comment:
Avoid using abstract classes to set behavior for classes when interfaces would suffice. Determine if 'Figure' needs to be an abstract class or if it could be an interface instead.